manifests/0000_08_cluster-dns-operator_02-deployment.yaml

```diff
@@ -15,6 +15,7 @@ spec:
     spec:
       dnsPolicy: Default
       nodeSelector:
+        beta.kubernetes.io/os: linux
         node-role.kubernetes.io/master: ''
```

Comment:
Although I approved this change to run dns operator on master :( I'm not clear if this is a requirement or it is needed to work properly in some environments?
@derekwaynecarr @ironcladlou ^^